Data/models/DataController.swift

```diff
@@ -44,6 +40,7 @@ public class DataController: NSObject {
                 }
             } catch {
                 log.error(error)
+                throw error
```

Comment:
Why are we re-throwing it? Is it caught somewhere?

Reply:
Being caught here: https://github.com/brave/brave-ios/pull/1555/files#diff-55d057fb46ea7e23daff9e66b489fdffR34
But don't think we have reason to catch it 2x.